app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] == "cp" and len(command_list) == 3:
```

Comment:
```suggestion
    if len(command_list) == 3 and command_list[0] == "cp":
```
Order make sense because if command_list will be empty it will give an error